extensions/blocks/podcast-player/edit.js

```diff
@@ -203,20 +219,55 @@ const PodcastPlayerEdit = ( {
 						onChange={ value => setAttributes( { showEpisodeDescription: value } ) }
 					/>
 				</PanelBody>
+				<PanelColorSettings
+					title={ __( 'Color Settings', 'jetpack' ) }
+					colorSettings={ [
+						{
+							value: primaryColorProp.color,
+							onChange: setPrimaryColor,
+							label: __( 'Primary Color', 'jetpack' ),
+						},
+						{
+							value: secondaryColorProp.color,
+							onChange: setSecondaryColor,
+							label: __( 'Secondary Color', 'jetpack' ),
+						},
+						{
+							value: backgroundColorProp.color,
+							onChange: setBackgroundColor,
+							label: __( 'Background Color', 'jetpack' ),
+						},
+					] }
+				>
+					<ContrastChecker
+						{ ...{
+							// Text is considered large if font size is greater or equal to 18pt or 24px,
+							// currently that's not the case for button.
```

Comment:
Is the button reference meant to be here or is it just copied from somewhere?

Reply:
copied from elsewhere, product if a wrong rebase I guess 🤦‍♂ . Updating

Reply:
but the comment there makes sense jus in case.